components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -881,6 +881,19 @@ public IdentityProvider getIdPById(String id, String tenantDomain,
         return identityProvider;
     }
 
+    /**
+     * Returns extended IDP with resource ID.
+     * Note: The UserDefinedFederatedAuthenticatorConfig object in the IdentityProvider is not serializable using
+     *       org.apache.commons.lang3.SerializationUtils, which is used in the authentication framework to clone the
+     *       authentication context. Hence, use getSerializableIdPByResourceId(String, String) in FrameworkUtils,
```

Comment:
Add @link annotation and this is not in FrameworkUtils right

Reply:
Cannot use link as we need to import framework component here.